common/app/views/support/Profile.scala

```diff
@@ -24,7 +24,7 @@ sealed trait ElementProfile {
     maybeAsset.flatMap(_.url).map(ImgSrc(_, this))
 
   def bestFor(image: ImageMedia): Option[ImageAsset] = {
-    if(ImageServerSwitch.isSwitchedOn) {
+    if(!ImageServerSwitch.isSwitchedOn) {
```

Comment:
what about switching the logic inside the `if else` rather than using the negation